### PR TITLE
Enhanced the mod's credits

### PR DESCRIPTION
**Pull request by OcelotWalrus**
* standardized the licenses writing (`CC-BY` licenses were written with and without dashes throughout the text: fixed that)
* fixed some errors that appear to be typos
* and some phrasing reformulations

### CODE REVIEW

File: mods/hv/CREDITS [hunk 1]
```diff
@@ -52,4 +52,7 @@ Using DiffPlex by Matthew Manela distributed
 under the Apache 2.0 license.
 
 Special thanks to the OpenRA team. Without
-them this mod would not have been possible.
+them, this mod wouldn't have existed.
+
+Finally, thanks to everyone on GitHub
+who contributed to the mod.
```
Comment (Mailaender) [inline mods/hv/CREDITS:58]: There were people who contributed without using GitHub, so please remove that line. It is also redundant to the credits above.

File: mods/hv/CREDITS [hunk 2]
```diff
@@ -19,28 +19,28 @@ Also thanks to:
...
-Includes sound files created by the US Gov.
-which are in the Public Domain.
+Includes sound files created by the US
+Gov. which are in the Public Domain.
 
 Also includes sounds made by Daniel Simon,
 Marvin, Mike Koenig, GunGuru, WEL, newlocknew,
 tutarap, twisterman, szegvari, cgeffex,
-dudeawesome, autistic-lucario
+dudeawesome and autistic-lucario
```
Comment (Mailaender) [inline mods/hv/CREDITS:39]: This list is probably not extensive anymore.

Comment (OcelotWalrus) [reply mods/hv/CREDITS:39]: Okay? So I keep it as I modified it?

Comment (Mailaender) [reply mods/hv/CREDITS:39]: Update from https://github.com/OpenHV/OpenHV/tree/main/mods/hv/bits/audio/cc-by-30#readme

**Pull request merged**